edx_name_affirmation/tasks.py

```diff
@@ -187,35 +193,35 @@ def proctoring_update_verified_name_task(
     bind=True, autoretry_for=(Exception,), default_retry_delay=DEFAULT_RETRY_SECONDS, max_retries=MAX_RETRIES,
 )
 @set_code_owner_attribute
-def delete_verified_name_task(self, idv_attempt_id, proctoring_attempt_id):
+def delete_verified_name_task(self, platform_verification_attempt_id, proctoring_attempt_id):
     """
     Celery task to delete a verified name based on an idv or proctoring attempt
     """
     # this case shouldn't happen, but should log as an error in case
-    if (idv_attempt_id and proctoring_attempt_id) or (not idv_attempt_id and not proctoring_attempt_id):
+    if (proctoring_attempt_id, platform_verification_attempt_id).count(None) != 1:
         log.error(
-            'A maximum of one attempt id should be provided for either a proctored exam attempt or IDV attempt.'
+            'A maximum of one attempt id should be provided'
         )
         return
 
     log_message = {'field_name': '', 'attempt_id': ''}
 
-    if idv_attempt_id:
-        verified_names = VerifiedName.objects.filter(verification_attempt_id=idv_attempt_id)
-        log_message['field_name'] = 'verification_attempt_id'
-        log_message['attempt_id'] = idv_attempt_id
+    if platform_verification_attempt_id:
+        verified_names = VerifiedName.objects.filter(platform_verification_attempt_id=platform_verification_attempt_id)
+        log_message['field_name'] = 'platform_verification_attempt_id'
+        log_message['attempt_id'] = platform_verification_attempt_id
     else:
         verified_names = VerifiedName.objects.filter(proctored_exam_attempt_id=proctoring_attempt_id)
         log_message['field_name'] = 'proctored_exam_attempt_id'
         log_message['attempt_id'] = proctoring_attempt_id
 
     if verified_names:
-        log.info(
+        log.info(  # there's a bug in this log message
```

Comment:
What's the bug? Is this something we need to ticket to fix or just a note-to-self that should be removed?

Reply:
ah this was a note to me to figure out if I wanted to fix this or ticket. The log message assumes the field the attempt_id is `verification_attempt_id` (now updated to platform_verification) but it could be any of those types. It's minor, given our timeline I'll just remove the comment and backlog this